packages/by-name/peerpod-imds-nat/setup-nat-for-imds.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# Copyright 2024 Edgeless Systems GmbH
+# SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
Should keep upstream copyright. Can we consume from cloud-api-adaptor.src instead of vendoring? Maybe even put it in cloud-api-adaptor.passthru, where we already have some other scripts from the repo.